less/index.less

```diff
@@ -4,56 +4,313 @@
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html,
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* MIXINS */
+@main-color: darkblue;
+@title-color: #211145;
+@font-color: white;
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+  /*  background-color:red; */
+}
+
+html,
+body {
+  height: 100%;
+  background-color: white;
+}
+.container {
+  max-width: 800px;
+  margin: auto;
+
+  h1 {
+    margin-top: 2rem;
+    text-align: center;
+    font-size: 3rem;
+  }
+}
+
+.nav-bar {
+  display: flex;
+  margin-top: 2rem;
+  background: @title-color;
+  color: white;
+
+  a {
+    margin: auto;
+  }
+
+  a:hover {
+    color: red;
+  }
+
+  /* NAV BAR MEDIA QUERY */
+  @media (max-width: 500px) {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    background: @title-color;
+
+    a {
+      padding-bottom: 1rem;
+      padding-top: 1rem;
+      border: 2px solid black;
+      width: 100%;
+      text-align: center;
+    }
+  }
+}
+
+.intro {
+  display: flex;
+  border: 2px solid black;
+  margin-top: 2rem;
+  background: @main-color;
+  color: @font-color;
+
+  img {
+    border: 2px solid black;
+    margin: 20px;
+  }
+
+  p {
+    margin-top: 3rem;
+  }
+}
+
+.qualifications {
+  border: 2px solid black;
+  margin-top: 2rem;
+  display: flex;
+  flex-display: row;
```

Comment:
The property is `flex-direction`, like so: `flex-direction: row;`. That (`row`) is also the default, so it's not necessary (and is probably why you didn't catch it before!)